packages/nextjs/components/Logo.tsx

```diff
@@ -8,24 +8,21 @@ type Props = React.ComponentProps<"svg"> & {
 /**
  * Lisk logo.
  */
-export const Logo: React.FC<Props> = ({ size = 32, ...props }) => {
+export const Logo: React.FC<Props> = ({ color = "#fff", size = 32 }) => {
```

Comment:
Please update this to #FFFFFF for consistency. Also spread the rest of the props since the component also expects svg props.